doc/content/sqa/v_and_v/materials/isoil/pressure_dependency.md

```diff
@@ -0,0 +1,131 @@
+
+!table id=strength-cases caption=Five combinations of $a_{0}$, $a_{1}$, $a_{2}$, and $p_{ref}$ used to investigate their effect on [p-ref-strength].
+| Case No. | $a_{0}$ | $a_{1}$ | $a_{2}$ | $p_{ref}$ |
+| - | - | - | - | - |
+| 1 | 0 | 0 | 1 | 6072.86 Pa |
+| 2 | 0 | 1 | 0 | 6072.86 Pa |
+| 3 | 1 | 0 | 0 | 6072.86 Pa |
+| 4 | 0 | 1 | 0 | 29430 Pa |
+| 5 | 0 | 1 | 0 | 42043 Pa |
+
+#### Results
+
+The engineering shear stress-strain backbone curve results for Cases 1, 2, and 3 are presented in [strength-comparison1], and those for Cases 2, 4, and 5 are presented in [strength-comparison2]. Both figures show their results along with the user-defined backbone curve shown in [p-ref-backbone-input]. The backbone curves were obtained in MOOSE by sampling the data during the initial load cycle, i.e., from 3 seconds to 4 seconds (see [#problem-statement]). It can be seen in [strength-comparison1] and [strength-comparison2] that MASTODON effectively modifies the shear strength while holding the shear modulus for a particular strain increment equal to that defined by the original backbone curve.
+
+!media media/materials/isoil/pressure_dependent_strength1.png
+       id=strength-comparison1
+       caption=The user-defined backbone curve and the engineering shear stress-strain curve in the ZX plane output by MASTODON during the initial loading cycle (from 3 seconds to 4 seconds) for for pressure dependent strength Cases 1, 2, and 3.
+
+!media media/materials/isoil/pressure_dependent_strength2.png
+       id=strength-comparison2
+       caption=The user-defined backbone curve and the engineering shear stress-strain curve in the ZX plane output by MASTODON during the initial loading cycle (from 3 seconds to 4 seconds) for pressure dependent strength Cases 2, 4, and 5.
+
+Since setting $a_{2}=1$ and $a_{1}=0$ causes the shear strength to increase exponentially, while setting $a_{2}=0$ and $a_{1}=1$ causes it to increase only linearly, [strength-comparison1] shows that Case 1 is significantly stronger than Case 2. Also, since setting $a_{0}=1$ while setting the others to zero causes the right-hand-side of [p-ref-strength] to be equal to unity, [strength-comparison1] shows that Case 3 is practically equivalent to the original backbone curve (the slight discrepancy arises from the presence of initial stresses, the tensile cutoff value, $p_{0}=-1$, and the application of the body force). Cases 4 and 5 are essentially Case 2 except with different $p_{ref}$ values, hence, Case 2 was included in [strength-comparison2] for comparison. This plot shows that setting the reference pressure equal to the applied vertical pressure (Case 4) causes the shear strength to remain virtually unmodified, while setting it lower (Case 2) causes the shear strength to increase and setting it higher (Case 5) causes the shear strength to decrease. These observations made from each of the five cases are arithmetically consistent with [p-ref-strength].
```

Comment:
'Since setting $a_{2}=1$ and $a_{1}=0$ causes the shear strength to increase exponentially, while setting $a_{2}=0$ and $a_{1}=1$ causes it to increase only linearly,' 

This is unclear/inaccurate -- case 1 (a2=1 and others zero) makes the strength to increase linearly with pressure since there is a square root. In case 2, it is proportional to the sq root of the pressure and in case 3 it is independent of the pressure. Please recheck the sentence,.